src/main/java/seedu/address/ui/InterviewCard.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.ui;
+
+import java.time.format.DateTimeFormatter;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.applicant.Applicant;
+import seedu.address.model.interview.Interview;
+
+/**
+ * An UI component that displays information of a {@code Interview}.
+ */
+public class InterviewCard extends UiPart<Region> {
+
+    private static final String FXML = "InterviewListCard.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     */
+
+    public final Interview interview;
+
+    @FXML
+    private HBox cardPane;
+    @FXML
+    private Label id;
+    @FXML
+    private Label date;
+    @FXML
+    private Label role;
+    @FXML
+    private Label name;
+    @FXML
+    private Label interviewphone;
+    @FXML
+    private Label interviewemail;
+
+    /**
+     * Creates a {@code InterviewCard} with the given {@code Interview} and index to display.
+     */
+    public InterviewCard(Interview interview, int displayedIndex) {
+        super(FXML);
+        this.interview = interview;
+
+        id.setText(displayedIndex + ". ");
+
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("EEEE dd MMM yyyy, HH:mma");
```

Comment:
Would it be better if `DateTimeFormatter formatter` is a static object?
E.g. 
```java
private static final DateTimeFormatter formatted = DateTimeFormatter.ofPattern("EEEE dd MMM yyyy, HH:mma")
```
This prevents reinitialization of formatter everything the constructor is called.